### DISCUSSION TITLE
Tsfresh input feature types

### DISCUSSION THREAD

**Discussion by monika2102** [blue-yonder/tsfresh#925] [Q&A]
can we use tsfresh after one hot encoding the categorical data for extracting the features and how do their significance can be judged

**Comment by kempa-liehr** [comment]
The FeatureSelection() transformer works with continuous, ordinal, and one-hot encoded categorical data and can be used independently from time-series feature extraction. 
Have a look at our white paper, which explains the feature selection in more detail: https://arxiv.org/abs/1610.07717
